distributed/chaos.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
```

Comment:
Can't wait to see what else ends up in this file :)

Reply:
Yup.  I almost put this into plugins.py, but decided that we should go boldly in this direction.  I'm looking forward to it.